docs/book/src/user/quick-start.md

```diff
@@ -329,6 +329,11 @@ Please visit the [Hetzner project][Hetzner provider].
 
 Please visit the [Metal3 project][Metal3 provider].
 
+{{#/tab }}
+{{#tab Nutanix}}
+
+Please follow the Cluster API Provider for [Nutanix Getting Started Guide](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/blob/main/docs/getting_started.md)
```

Comment:
nit

The guide at this link should be updated to reference 0.2.2 or 0.2.1. It looks like 0.2.0 doesn't exist so the commands don't work.

Reply:
let me fix it , doc is not yet dynamic and i was so impatient to push this PR :-D

Reply:
doc ix fixed thanks @killianmuldoon